Vizzuality/heco-invest
# Ensure all requests are localised on the server and client

This PR makes sure that:
- All the requests initiated on the server are localised
- All the requests initiated on the server are refetched (and localised) on the client when the locale changes

## Testing instructions

- Choose a non default locale such as Portuguese
- Open a SSR-ed page such as a project's page and check that the correct locale was sent to the API (you can log it)
- Change the locale on that same page and check that all the queries are refetched with the new locale

## Tracking

- [LET-619](https://vizzuality.atlassian.net/browse/LET-619)
- [LET-620](https://vizzuality.atlassian.net/browse/LET-620)
- [LET-621](https://vizzuality.atlassian.net/browse/LET-621)
